CHANGELOG-v3.adoc

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+== AM - 3.21.3 (2023-07-07)
```

Comment:
Are we missing the performance work we have done?

Reply:
It has been filtered out by the script because it's a jira private issue. Do we want to specifically add something for this?

Reply:
@ben-gravitee ?

Reply:
That's ok @mariano-fernandez